editor/editor_node.cpp

```diff
@@ -166,6 +166,7 @@
 #include "editor/themes/editor_theme_manager.h"
 #include "editor/window_wrapper.h"
 
+#include "editor_script.h"
```

Comment:
```suggestion
#include "editor/editor_script.h"

```
Although you'll need to sort it (clang-format can do that).